feat(changelog): containers-added-serverless-containers-custom-health-ch 2025-02-26 (#4503)
scaleway/docs-content: changelog/february2025/2025-02-26-containers-added-serverless-containers-custom-health-ch.mdx

---

> title: Serverless Containers Custom health checks

Reviewer: ```suggestion
title: Serverless Containers custom health checks
```

status: added
date: 2025-02-26
category: serverless
product: containers
---

> This feature allows users to define [custom healthcheck](https://www.scaleway.com/en/docs/serverless-containers/concepts/#health-check) for their Serverless Containers, enabling more precise monitoring and improved reliability.

Reviewer: ```suggestion
This feature allows you to define [custom health checks](https://www.scaleway.com/en/docs/serverless-containers/concepts/#health-check) for your Serverless Containers, enabling more precise monitoring and improved reliability.
```
